__tests__/app.test.js

```diff
@@ -122,4 +122,12 @@ describe("Patch votes",()=>{
             expect(responce.body.article.votes).toBe(110)
         })
     })
+})
+
+describe("DELETE comment",()=>{
+    test("Deletes comment at endpoint and sends back 204.",()=>{
```

Comment:
Your test descriptions are so much clearer now! 👌  Really nice to see.
A convention to try for descriptions in your test suite, is to have the `describe` block title as the endpoint path, and then the test description will have the method and status code stated and then a description the response body.

E.g. for this one:
```
describe("/api/comment/:comment_id",()=>{
    test("DELETE 204: Deletes comment and send back an empty response body",()=>{
```